src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Rows.component.tsx

```diff
@@ -1,27 +1,33 @@
 /** @jsx jsx */
-import { jsx, SerializedStyles } from "@emotion/core";
+import { css, jsx, SerializedStyles } from "@emotion/core";
 import TableCell from "@material-ui/core/TableCell";
 
-function chooseAlignment(element: string): "left" | "right" {
+function checkIfElementIsName(element: string): ["left" | "right", boolean] {
     if (element === "name") {
-        return "left";
+        return ["left", true];
     }
-    return "right";
+    return ["right", false];
 }
 
+const tableCellStyle = (isName: boolean): SerializedStyles => css`
+    box-sizing: border-box;
+    border-right: 1px solid lightgrey;
+    :last-child {
+        border-right: none;
+    }
+    width: ${isName ? "160px" : "auto"};
+    min-width: ${isName ? "160px" : "auto"};
+`;
+
 export interface TableContentRowsProps {
     row: Record<string, string>;
     classes: Record<"tableCell", string>;
-    isRowAndCol: boolean;
-    style: (isRow: boolean, isRowAndCol: boolean) => SerializedStyles;
 }
 
 /**
  * @desc Returns list of table cells for one table row
  * @param {Record<string, string>} row - object with the values for the row
  * @param {Record<"tableCell", string>,} classes - material-ui styling of one table cell
- * @param {boolean} isRowAndCol - true if row and column is selected
- * @param {(isRow: boolean, isRowAndCol: boolean) => SerializedStyles} - style of the row

```

Comment:
same here, you should move the description to the props interface

Reply:
Done.